setup.cfg

```diff
@@ -0,0 +1,2 @@
+[pep8]
```

Comment:
It's just to run Python's `pep8` style-guide checker (which I run manually) but tell it to ignore 4-spaces style since all wasm codebases seem to use 2-spaces. I mostly just care about consistency, we could switch all the code to 4-spaces and then `pep8` would be happy.